dockstore-webservice/pom.xml

```diff
@@ -993,9 +843,13 @@
         </dependency>
         <dependency>
             <groupId>org.orcid</groupId>
-            <artifactId>orcid-model</artifactId>
```

Comment:
In particular, this one is now our own fork (which we'll propose as a PR back to orcid after our testing)